### PR TITLE
Updated macOS installation steps with homebrew

### PR DESCRIPTION
**Pull request by abhi3700**
## Description

This PR is to replace the installation steps with homebrew for macOS in README.

There was an issue: https://github.com/subspace/pulsar/issues/270 which is resolved with https://github.com/subspace/homebrew-pulsar/pull/1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,30 +107,11 @@ sudo apt-get install llvm clang cmake
...
-3. Activate the changes:
-
-```bash
-source ~/.zshrc
-# or
-source ~/.bashrc
-```
-
-4. Verify that `llvm` is installed:
+Install via Homebrew:
```
Comment (nazar-pc) [inline README.md:110]: This is incorrect change. This describes how to build software from source, not how to install precompiled software using homebrew.

Also while updating this is fine, the user-facing docs are at https://docs.subspace.network/

Comment (abhi3700) [reply README.md:110]: Understood!

Ok I will revert the change.

Then should we have a another section with homebrew installation for macOS under Binary in README? 

Like this:

```markdown
## Binary

### macOS

Install using homebrew:


```

Comment (nazar-pc) [reply README.md:110]: Yeah, I think that'll work.

cc @EmilFattakhov

Comment (abhi3700) [reply README.md:121]: Fixed. [commit](https://github.com/subspace/pulsar/pull/295/commits/3deb63475a8e40037ceca881fb4068c21d27576b)

**Comment by ParthDesai** [comment]
@abhi3700 Please rebase the PR. Right now the changes include almost entire repository.

**Pull request closed without merging**